apps/web/app/(app)/environments/[environmentId]/surveys/[surveyId]/edit/components/ColorSurveyBg.tsx

```diff
@@ -0,0 +1,66 @@
+import { useState } from "react";
+import { ColorPicker } from "@formbricks/ui/ColorPicker";
+import { Label } from "@formbricks/ui/Label";
+import { TSurvey } from "@formbricks/types/surveys";
+
+interface ColorSurveyBgBgProps {
+  localSurvey?: TSurvey;
+  handleBgChange: (bg: string, bgType: string) => void;
+}
+
+export default function ColorSurveyBg({ localSurvey, handleBgChange }: ColorSurveyBgBgProps) {
+  const colours = [
+    "#FFF2D8",
+    "#EAD7BB",
+    "#BCA37F",
+    "#113946",
+    "#04364A",
+    "#176B87",
+    "#64CCC5",
+    "#DAFFFB",
+    "#132043",
+    "#1F4172",
+    "#F1B4BB",
+    "#FDF0F0",
+    "#001524",
+    "#445D48",
+    "#D6CC99",
+    "#FDE5D4",
+    "#BEADFA",
+    "#D0BFFF",
+    "#DFCCFB",
+    "#FFF8C9",
+    "#FF8080",
+    "#FFCF96",
```

Comment:
can you move these colors to the constatns file so that in future if we want to add/change or reuse any colors, we can directly refer the constants file, improving the reusability.

Reply:
sure, makes sense, I'll make the changes, thanks